src/clib/lib/enkf/model_config.cpp

```diff
@@ -239,8 +239,7 @@ model_config_type *model_config_alloc_empty() {
     model_config->num_realizations = 0;
     model_config->obs_config_file = NULL;
 
-    model_config_set_enspath(model_config,
-                             fs::absolute(DEFAULT_ENSPATH).c_str());
+    model_config_set_enspath(model_config, DEFAULT_ENSPATH);
```

Comment:
This will probably cause the test `test_default_model_config_ens_path` to fail. 

So as a quick fix I would suggest also replacing 
```python
    if (config_content_has_item(config, ENSPATH_KEY))
        model_config_set_enspath(
            model_config,
            config_content_get_value_as_abspath(config, ENSPATH_KEY));
```
with 
``` python 
    if (config_content_has_item(config, ENSPATH_KEY))
        model_config_set_enspath(
            model_config,
            config_content_get_value_as_abspath(config, ENSPATH_KEY));
    else
        model_config_set_enspath(
            model_config,
            config_content_get_value_as_abspath(config, DEFAULT_RUNPATH));
```
I know this makes `model_config_set_enspath(model_config, DEFAULT_ENSPATH);` on ln 242 (what a nice palindromic number) a bit useless but it might be ok to just remove that line altogether

Reply:
also `namespace fs = std::filesystem;` can be removed also

Reply:
What if we instead fix the test to assert that `assert default_ens_path == set_in_file_ens_path` but where we add `res_config.config_path` to `default_ens_path`?

Reply:
I am not sure the test needs a fix. Are you suggesting fixing `getEnspath` from `ModelConfig` to make sure it contains `res_config.config_path` ?

Reply:
**Fixed** :arrow_up:  replaced `config_content_get_value_as_abspath(config, DEFAULT_RUNPATH));` with `config_content_get_value_as_abspath(config, DEFAULT_ENSPATH));`